mkosi/kmod.py

```diff
@@ -234,12 +238,27 @@ def process_kernel_modules(
                 p.rmdir()
 
         for fw in firmware:
+            # assume any files in /usr/lib/firmware/updates are required
+            if fw.is_relative_to(firmwared / "updates"):
+                continue
+
+            if fw in required and (root / fw).is_symlink():
+                # mopdules may depend on a symbolic link to the actual firmware file, so
```

Comment:
```suggestion
                # modules may depend on a symbolic link to the actual firmware file, so
```
there's also typos in the commit messages, `lnk` instead of `link` in the first and `an` instead of `and` in the second.